jenkins-build.sh

```diff
@@ -40,8 +42,14 @@ where:
 
 --skip_build             Do not run build step
 
+--skip_ut_build          Do not run build step for UTs
+
 --skip_tests             Do not run tests, exit before test run
```

Comment:
where --skip_tests option would be used now ?

Reply:
can we can remove this option now ?

Reply:
No.  `--skip_ut_run` and `--skip_st_run` will simply skip part of testing and proceed to tear down the running cluster.  While `--skip_tests` will actually terminate jenkins-build.sh at certain point, and will leave cluster in running state so you an run manual tests.  So we need all three.